app/javascript/vue/protocol/modals/reorderable_items_modal.vue

```diff
@@ -81,7 +81,7 @@
         return str.length > 56 ? str.slice(0, 56) + "..." : str;
       },
       nameWithFallbacks(item) {
-        return item.attributes.name || this.getTitle(item);
+        return this.getTitle(item) || item.attributes.name;
```

Comment:
This seems incorrect, as the fallback title will always get called, even if `item.attributes.name` is available.

Reply:
it will get called, but will return false, since if item.attribute.name exists, the function returns empty string, hence the logic goes to item.attribute.name itself